docs/pages/quickstart.rst

```diff
@@ -28,7 +28,7 @@ Habitat is a platform for embodied AI research that consists of:
     :gh:`[github-repo] <facebookresearch/habitat-api>`
```

Comment:
@mosra Seems the :gh: macro is broken

Reply:
Sorry, but this sentence is *absolutely* unhelpful. *How* is it broken? Is the link not working? Is the title wrong? What do you expect to see instead?

I can't see any problem here. The quickstart originally had the `[github-repo]` links in this exact form, so when converting that to m.css, I preserved that.